Etabs_Adapter/Helpers/Results.cs

```diff
@@ -244,6 +246,24 @@ public static List<BarForce> GetBarForce(cSapModel model, IList ids = null, ILis
             //Get out loadcases, get all for null list
             loadcaseIds = CheckAndGetCases(model, cases);
 
+            for (int i = 0; i < cases.Count; i++)
```

Comment:
Think the additional checks added in here makes sense, but I would move them to the `CheckAndGetCases` method so that they can be used by all force extraction scripts!